packages/frontend-shared/src/gql-components/RecordRunModal.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <StandardModal
+    class="transition transition-all duration-200"
+    variant="bare"
+    :title="t('specPage.banners.record.title')"
+    :model-value="isModalOpen"
+    :help-link="helpLink"
+    :no-help="!helpLink"
+    data-cy="record-run-modal"
+    @update:model-value="close"
+  >
+    <div class="max-w-175 py-7 px-6 text-gray-600">
+      <RecordPrompt />
+    </div>
+  </StandardModal>
+</template>
+
+<script setup lang="ts" >
+import { useI18n } from '@cy/i18n'
+import StandardModal from '../components/StandardModal.vue'
+import RecordPrompt from './RecordPrompt.vue'
+import { getUtmSource } from '../utils/getUtmSource'
+
+const { t } = useI18n()
+
+const props = defineProps<{
+  isModalOpen: boolean
+  close: () => void
+  utmMedium: string
+  utmContent?: string
+}>()
+
+const helpLink =
+`https://on.cypress.io/cypress-run-record-key?utm_medium="${props.utmMedium}",utm_source="${getUtmSource()}"${props.utmContent ? `,utm_content="${props.utmContent}"` : ''}`
```

Comment:
Are we sure about the formatting of this? Should the query params really be separated by `,` instead of `&`? Why not use the existing `getUrlWithParams` utility function to build this?

Reply:
Oh wow, yeah having `,` as the separator was a mistake. I didn't see that there was a utility function, will definitely use that 👍🏻